src/main/java/com/automationpractice/pageobjects/ContactUsPage.java

```diff
@@ -0,0 +1,83 @@
+package com.automationpractice.pageobjects;
+
+import com.automationpractice.BasePage;
+import com.utils.ReadPropertyFile;
+import io.qameta.allure.Step;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import java.io.File;
+
+public class ContactUsPage extends BasePage {
+
+    private WebDriver driver;
+    String baseUrl = ReadPropertyFile.getInstance().getContactUsPageUrl();
+    Logger log = Logger.getLogger("applicationLogger");
+
+    /*Constructor of current ContactUsPage class*/
+    public ContactUsPage(WebDriver driver) {
+        this.driver = driver;
+        PageFactory.initElements(driver, this);
+    }
+
+    @FindBy(xpath = "//h1[contains(@class, 'page-heading')]")
+    private WebElement contactUsPageHeading;
+
+    @FindBy(xpath = "//select[@id='id_contact']")
+    private WebElement subjectDropDownContactUsPage;
+
+    @FindBy(xpath = "//p[contains(@class, 'desc_contact') and not(contains(@style, 'display: none'))]")
+    private WebElement subjectDropDownTitleAfterOptionSelectionContactUsPage;
+
+    @FindBy(xpath = "//input[@type='file']")
+    private WebElement chooseFileContactUsPage;
+
+    @FindBy(xpath = "//span[@class='filename']")
+    private WebElement uploadedFileName;
+
+    @Step("Open Contact Us page")
+    public ContactUsPage open() {
+        log.debug("Opening Contact Us page");
+        driver.get(baseUrl);
+        return this;
+    }
+
+    @Step("Check title on Contact Us page")
+    public String getContactUsPageHeading() {
+        log.debug("Checking title on Contact Us page");
+        return contactUsPageHeading.getText();
+    }
+
+    @Step("Select option in Subject Heading drop down")
+    public ContactUsPage selectOptionInSubjectHeadingDropDown(String option) {
+        log.debug("Select option in Subject Heading drop down");
+        String optionInSubjectDropDownContactUsPage = String.format("//select[@id='id_contact']/option[text()='%s']", option);
+        subjectDropDownContactUsPage.click();
+        driver.findElement(By.xpath(optionInSubjectDropDownContactUsPage)).click();
```

Comment:
It would be better to extract this into a method under the FyndBy elements
This will allow you to re-use element and store all the elements and locators in the same document arae